applications/management/commands/assign_application_area_type.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+
+from django.core.management import BaseCommand
+
+from applications.enums import ApplicationAreaType
+from applications.models import WinterStorageApplication
+
+
+class Command(BaseCommand):
+    help = "Assign Application area_type for existing applications"
+
+    def handle(self, *args, **options):
+        # Get ws applictions created after 14.9.2020
+        new_applications = WinterStorageApplication.objects.filter(
+            created_at__gt=date(2020, 9, 14)
+        )
+
+        for application in new_applications:
+            application.area_type = application.resolve_area_type()
+            application.save()
+
+        # Get ws applictions created before 14.9.2020
+        old_applications = WinterStorageApplication.objects.filter(
+            created_at__lt=date(2020, 9, 14)
+        )
+
+        for application in old_applications:
+            # Mark all older applications as MARKED
+            application.area_type = ApplicationAreaType.MARKED
+            application.save()
```

Comment:
and since here you simply set the same value for every object, you could do:
```python
WinterStorageApplication.objects.filter(
    created_at__lt=date(2020, 9, 14)
).update(area_type=ApplicationAreaType.MARKED)
```

Reply:
Good catch, done.